packages/palette/src/elements/index.tsx

```diff
@@ -1,4 +1,5 @@
 export * from "./Avatar"
+export * from "./AspectRatioBox"
```

Comment:
soooooo minor -- this should go before avatar

Reply:
omg